pkg/auth/cell.go

```diff
@@ -33,13 +33,13 @@ var Cell = cell.Module(
 		newSignalRegistration,
 		// Null auth handler provides support for auth type "null" - which always succeeds.
```

Comment:
it shouldn't be possible to use the null auth handler (always pass) -> should be removed from the cell

Reply:
good point :100: